cypress/integration/seededFlows/homeFeedFlows/feedNavigation.spec.js

```diff
@@ -87,6 +95,14 @@ describe('Home Feed Navigation', () => {
   });
 
   it('should navigate to Infinity view', () => {
+    cy.findByRole('navigation', { name: 'View posts by' }).within(() => {
+      cy.findByRole('link', { name: 'Top' }).as('top');
+      cy.get('@top').click();
+    });
+
+    // this url check serves to wait for the page transition
+    cy.url().should('contain', '/top/week');
```

Comment:
it might be sufficient to just look for url contains `/top/` and leave out `/week` (for all of these checks after clicking 'Top'.